tests/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-find_package(Boost 1.55.0 REQUIRED)
+# We move the test files into the testing directory
+file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/hypervolume_test_data/" DESTINATION "${CMAKE_BINARY_DIR}/tests/hypervolume_test_data/")
```

Comment:
The thing is if I do it on arch then:

-- /home/dario/Documents/pagmo2/build
-- /home/dario/Documents/pagmo2/build

they are the same!! But when I did change it appveyor failed complaining the files where not found.